utils/physics_utilities.py

```diff
@@ -0,0 +1,95 @@
+import re
+
+
+def pdg_id(particle):
```

Comment:
Maybe at this stage there is only one function in this script and it can be moved to `jet_variables` where it is only used.

Reply:
moved to jet_variables